space_packet_parser/packets.py

```diff
@@ -29,13 +30,67 @@ class SequenceFlags(IntEnum):
 
 
 class RawPacketData(bytes):
-    """A class to represent raw packet data as bytes but whose length is represented by bit length.
+    """A class containing the raw binary packet data.
 
     This class is a subclass of bytes and is used to represent the raw packet data
-    in a more readable way. It is used to store the raw packet data in the Packet
+    in a more readable way. It is used to store the binary data in the Packet
     class and used to keep track of the current parsing position (accessible through the `pos` attribute).
+    """
+    pos = 0  # Current position in bits
+
```

Comment:
Did you want to give `RawPacketData` a `__str__` dunder like `CCSDSPacketBytes`? If we follow the inheritance suggestion below, we could just define it once in the (this) parent class.